cardano_cli_helper.py

```diff
@@ -32,8 +32,8 @@ def getCardanoCliValue(command, key):
         stderr = stderr.decode("utf-8")
         print(stdout)
         print(stderr)
-        if not stderr == '':
-            return (-1)
+        if process.returncode != 0:
+            raise Exception(f'Error calling {command}')
```

Comment:
No sure this change won't affect other part of the code some maybe you want to revert it. Basically the problem was that the register SPO script would continue running while it was failing. And I did not have the failing command to be able to understand what was wrong.